src/utils/composeStores.js

```diff
@@ -1,8 +1,17 @@
 import mapValues from '../utils/mapValues';
 import pick from '../utils/pick';
+import invariant from 'invariant';
 
 export default function composeStores(stores) {
   const finalStores = pick(stores, (val) => typeof val === 'function');
+
+  Object.keys(finalStores).forEach(key => {
+    invariant(
+        typeof finalStores[key]({}, {}) !== 'undefined',
```

Comment:
If so, then i would omit 'undefined' and change message to something like "Your %s Store must return the state given to it for any unknown actions."